apps/moodlenet/templates/services/backend/_env.yml.j2

```diff
@@ -0,0 +1,54 @@
+# server domain name:
+HOSTNAME: "{{ moodlenet_host }}"
+
+# full backend URL - server protocol, domain name, and port
+# (if it isn't the default for protocol like 80 or 443):
+BASE_URL: "https://{{ moodlenet_host }}"
+
+# cc public activities to the mothership for indexing
+CONNECT_WITH_MOTHERSHIP: "{{ moodlenet_mothership_connect }}"
+
+# which mothership environment to search (defaults to a test index, change this in production)
+REACT_APP_MOTHERSHIP_ENV: "moodlenet_mothership_next"
+
+# a tagline for your instance
+INSTANCE_DESCRIPTION: "{{ moodlenet_instance_description }}"
+
+# whether to require an email address to first be invited (or access-listed with MoodleNet.Access) before being able to sign up
+INVITE_ONLY: "{{ moodlenet_invite_only }}"
+
+# whether to automatically alter the database schema when you upgrade the app
+AUTORUN_DB_MIGRATIONS: "true"
+
+LANG: "{{ moodlenet_lang }}"
```

Comment:
missing description. You were telling us about the notion of default language

Reply:
Thanks, I added a description.

(Note that for the moment no translation are shipped with the backend image)